dlt/common/schema/schema.py

```diff
@@ -240,6 +240,15 @@ def coerce_row(
                         updated_table_partial["columns"] = {}
                     updated_table_partial["columns"][new_col_name] = new_col_def
 
+        # insert columns defs for scd2 (TODO: where to do this properly, maybe in a step after the normalization?)
```

Comment:
where is the right place to update the schema based on destination settings? maybe we should do this at the beginning of the load step? I am not sure.